src/context/calculation/calculateCostBasis.test.js

```diff
@@ -180,4 +180,49 @@ describe("test calculateCostBasis function", () => {
       },
     ]);
   });
+
+  test("throws error if amount sold is greater than amount owned", () => {
+    const sellMoreThanOwnData = [
+      {
+        service: "Coinbase",
+        asset: "ETH",
+        transDate: "2018-01-31T07:01:31.000Z",
+        type: "Buy",
+        qty: 6,
+        amount: 600,
+        fee: 6,
+        id: "id1",
+        displayDate: "Wed, Jan 31, 2018 1:01 AM",
+      },
+      {
+        service: "Coinbase",
+        asset: "ETH",
+        transDate: "2018-03-31T07:01:38.000Z",
+        type: "Sell",
+        qty: 8,
+        amount: 600,
+        fee: 2,
+        id: "id2",
+        displayDate: "Wed, Jan 31, 2018 1:01 AM",
+      },
+    ];
+
+    let actualErrorObject = null;
+
+    try {
+      calculateCostBasis(sellMoreThanOwnData);
+    } catch (err) {
+      actualErrorObject = err;
+    }
+
+    expect(() => calculateCostBasis(sellMoreThanOwnData)).toThrowError(
```

Comment:
since we already have `actualErrorObject` stored, let's see if we can use that here to assert that it's instance of `SellMoreThanOwnError`. also there's a possibility, although slim, that this invocation will yield a different result than the first time it's called and stored in the variable